.gitlab-ci.yml

```diff
@@ -1,7 +1,7 @@
 stages:
-  - formatter
   - generator
   - test-containers
+  - format-check
```

Comment:
I thought having the format-check first would be best before running any tests ...

Reply:
if there's no format fixing, this is is better than reporting "hey the code ain't even formatted, I'm not gonna even look at it".

Reply:
but the idea is to test with the formatting applied, no? at least that's how I understand it

Reply:
yeah, with autoformatting it didn't make sense to run the tests on unformatted code because they would be run on the formatted either. Without it, it doesn't matter, and it's better to see whether tests fail than whether formatting is wrong.

Reply:
but then don't we have to test as well the formatted code with the same base image?